### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by Jeko0**

### CODE REVIEW

File: Pythagora/taks3.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+# task3
+class Pythagora
+  def initialize(a, b, c)
+    @sides = [a, b, c].sort
+    raise ArgumentError if @sides.any?(&:negative?)
+  end
+
+  def pythagorean?
+    @sides[2]**2 == @sides[0]**2 + @sides[1]**2
```
Comment (alexshvedko) [inline Pythagora/taks3.rb]: https://rubystyle.guide/#spaces-indentation
`@sides[2] ** 2 == @sides[0] ** 2 + @sides[1] ** 2`

File: Pythagora/taks3.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  end
+
+  def isoscele?
```
Comment (alexshvedko) [inline Pythagora/taks3.rb]: rename to `is_isoscele?`

File: Pythagora/taks3.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    @sides.any? { |side| @sides.count(side) == 2 }
+  end
+
+  def equilateral?
+    @sides.all? { |side| @sides.count(side) == 3 }
+  end
+
+  def calculate
+    if pythagorean?
```
Comment (alexshvedko) [inline Pythagora/taks3.rb:23]: where is else?

  if pythagorean?
      puts 'Triangle is pythagorean'
    elsif isoscele?
      puts 'Triangle is isoscele and rectangular'
    elsif equilateral?
      puts 'Triangle is equilateral but not rectangular'
    else
    
    end

File: Pythagora/taks3.rb (hunk 1 above)
Comment (Jeko0) [reply Pythagora/taks3.rb]: yes my bad
didn't saw

**Pull request closed without merging**